cli/lib/kontena/cli/stacks/search_command.rb

```diff
@@ -5,12 +5,14 @@ class SearchCommand < Kontena::Command
     include Kontena::Cli::Common
     include Common
 
-    parameter '[QUERY]', "Query string"
-
-    requires_current_account_token
+    parameter "[QUERY]", "Query string"
 
     def execute
-      puts stacks_client.search(query).inspect
+      results = stacks_client.search(query.to_s)
+      exit_with_error 'Nothing found' if results.empty?
+      results.each do |stack|
+        puts stack['name']
```

Comment:
Well, yes and no.

It would make sense that it would say 'stack'.  Currently it returns 'name'.